dj/ui/FPM.ftd

```diff
@@ -0,0 +1,10 @@
+-- import: fpm
+
+-- fpm.package: movie-db
+base: https://raw.githubusercontent.com/FifthTry/movie-db
+
+-- fpm.dependency: fifthtry.github.io/doc-site as ds
```

Comment:
We should not use doc-site for this project. Create a custom page please. Feel free to use dependencies of doc-site, eg header, typography etc, but create custom UI.